wiki/Beatmap/Pattern/Mania/Trills/en.md

```diff
@@ -0,0 +1,29 @@
+---
+stub: true
+tags:
+  - Trills
+---
+
+# Trills
+
+A **Trill** is two or more consecutive notes that alternate between two different columns. 
```

Comment:
Would then 1-4 be a trill? I think you need at least 4 notes to consider something a trill.

Reply:
Oops, read RC glossary wrong. Should be:
```suggestion
A **Trill** is three or more consecutive notes that alternate between two different columns. 
```